lte/gateway/deploy/roles/magma_deb/files/post_install_changes.sh

```diff
@@ -0,0 +1,66 @@
+mkdir -p /var/opt/magma/configs
+ln -sf /home/vagrant/magma/lte/gateway/configs/control_proxy.yml /var/opt/magma/configs/control_proxy.yml
+
+# Copy test certificates
+mkdir -p /var/opt/magma/certs/
+cp /home/vagrant/magma/.cache/test_certs/rootCA.* /var/opt/magma/certs/
+
+# Override pipelined and sessiond production configuration
+ln -sf /home/vagrant/magma/lte/gateway/configs/pipelined.yml /etc/magma/pipelined.yml
+ln -sf /home/vagrant/magma/lte/gateway/configs/sessiond.yml /etc/magma/sessiond.yml
+
+# Ensure changes in repository are used in services
+ln -sf /home/vagrant/magma/lte/gateway/configs/templates/mme.conf.template /etc/magma/templates/mme.conf.template
+ln -sf /home/vagrant/magma/lte/gateway/configs/gateway.mconfig /etc/magma/gateway.mconfig
+
+# ssh test configuration
+cp /home/vagrant/magma/lte/gateway/deploy/roles/dev_common/files/sshd_config /etc/ssh
+service ssh reload
+
+# Fake python environment so magma_test ssh calls on activate will not fail
+mkdir -p /home/vagrant/build/python/bin
+touch /home/vagrant/build/python/bin/activate
+
+# Make magma_test ssh calls on python3 possible
+ln -sf /usr/bin/python3 /home/vagrant/build/python/bin/python3
+
+# Override production service configurations with test configurations
+rm -f /etc/systemd/system/magma*
+cp /home/vagrant/magma/orc8r/tools/ansible/roles/gateway_services/files/magma.service /etc/systemd/system/magma@.service
+cp /home/vagrant/magma/lte/gateway/deploy/roles/magma/files/systemd/magma_connectiond.service /etc/systemd/system/magma@connectiond.service
+cp /home/vagrant/magma/orc8r/tools/ansible/roles/gateway_services/files/magma_control_proxy.service /etc/systemd/system/magma@control_proxy.service
+cp /home/vagrant/magma/lte/gateway/deploy/roles/magma/files/systemd/magma_ctraced.service /etc/systemd/system/magma@ctraced.service
+cp /home/vagrant/magma/lte/gateway/deploy/roles/magma/files/systemd/magma_dnsd.service /etc/systemd/system/magma@dnsd.service
+cp /home/vagrant/magma/lte/gateway/deploy/roles/magma/files/systemd/magma_envoy_controller.service /etc/systemd/system/magma@envoy_controller.service
+cp /home/vagrant/magma/lte/gateway/deploy/roles/magma/files/systemd/magma_eventd.service /etc/systemd/system/magma@eventd.service
+cp /home/vagrant/magma/lte/gateway/deploy/roles/magma/files/systemd/magma_liagentd.service /etc/systemd/system/magma@liagentd.service
+cp /home/vagrant/magma/lte/gateway/deploy/roles/magma/files/systemd/magma_lighttpd.service /etc/systemd/system/magma@lighttpd.service
+cp /home/vagrant/magma/lte/gateway/deploy/roles/magma/files/systemd/magma_magmad.service /etc/systemd/system/magma@magmad.service
+cp /home/vagrant/magma/lte/gateway/deploy/roles/magma/files/systemd/magma_mme.service /etc/systemd/system/magma@mme.service
+# TODO: GH13551. Ugly workaround to prevent service failures because of memory leaks 
+sed -i 's/\[Service\]/[Service]\nEnvironment="LSAN_OPTIONS=detect_leaks=0"/' /etc/systemd/system/magma@mme.service
```

Comment:
The workaround suppresses all memory leaks, which is not a good idea. It will be better to suppress only the two known issues in libfluid and libczmq, but highlight if new memory leaks come up.

Reply:
Thanks for the comment. In the end, we've decided to remove this line entirely since we've found that pipelined is failing even with it. We propose to merge this PR as a first attempt and then to have a more detailed look at this in future PRs (already planned), when we try to use this in the CI to execute integration tests. Does this sound okay to you?

Reply:
@voisey , do you plan to remove this line as part of this PR, or future PR?

Reply:
I've now removed it from this PR.